view/search/resource.phtml

```diff
@@ -36,14 +36,11 @@
 ?>
 
 <?php $escape = $this->plugin('escapeHtml'); ?>
+<?php $thumbnail = $this->plugin('thumbnail'); ?>
```

Comment:
Not important, but as it is used only once (and probably never will be used more than once), it's not really useful IMO to have it as a variable